doc/api/cli.md

```diff
@@ -1980,6 +1980,24 @@ On a machine with 2 GB of memory, consider setting this to
 $ node --max-old-space-size=1536 index.js
 ```
 
+### `--max_semi_space_size=SIZE` (in megabytes)
+
+Sets the maximum
+[semi\_space](https://www.memorymanagement.org/glossary/s.html#semi.space)
+size for V8's [scavenge](https://v8.dev/blog/orinoco-parallel-scavenger)
+garbage collector.
+Increasing the value of max\_semi\_space\_size may bring
+throughput improvement for Node.js and the cost is more
+memory consumption.
+The actual throughput improvement and memory consumption
+are relevant to your application
+([reference](https://github.com/nodejs/node/issues/42511)).
```

Comment:
I'm not sure I understand this sentence, do you mean it's relevant to any Node.js application, and if so, how do you know?

Reply:
I tuned the max_semi_space_size value and tested it on two workloads, this is the results:
![img](https://user-images.githubusercontent.com/88813722/160621733-140758a1-b172-4549-a181-e83fa7a79747.png)
From the figure we can see that the trend of throughput improve are similar on both workloads, but the absolute value are different. I just want the user to test and choose the optimal configuration for their own application.